tests/modelling/test_us_equity_pricing_loader.py

```diff
@@ -97,7 +97,7 @@
 class BcolzDailyBarTestCase(TestCase):
 
     def setUp(self):
-        all_trading_days = TradingEnvironment.instance().trading_days
+        all_trading_days = TradingEnvironment().trading_days
```

Comment:
Agreed - I will do that where I can. 

There are some tests where the intention is to build a new TradingEnvironment so that there will be new/empty Assets, so some of them may not be able to have an env built at class level, but I'll see which of those can.